src/locutus_util/helpers.py

```diff
@@ -0,0 +1,64 @@
+import subprocess
+from google.cloud import firestore
+import time
+
+def update_gcloud_project(project_id):
+    """Update the active Google Cloud project."""
+    command = ["gcloud", "config", "set", "project", project_id]
+    
+    try:
+        print(f"Updating Google Cloud project to: {project_id}")
+        subprocess.run(command, check=True)
+        print(f"Project updated successfully. {project_id}")
+    except subprocess.CalledProcessError as e:
+        print(f"Error updating project: {e}")
+
+
+def delete_document(collection_name, document_id):
+    """Delete a document and any docuements in their subcollections(removes the 
+    subcollection), recursively."""
+    db = firestore.Client()
+    doc_ref = db.collection(collection_name).document(document_id)
+
+    # Iterate through all subcollections of the document
+    for subcollection in doc_ref.collections():
+        subcollection_name = subcollection.id
+        print(f"Deleting subcollection '{subcollection_name}' for document '{document_id}'.")
+
+        # Delete the subcollection documents
+        for subdoc in subcollection.stream():
+            print(f"Deleting subdocument '{subdoc.id}' from subcollection '{subcollection_name}'.")
+            delete_document(f"{collection_name}/{document_id}/{subcollection_name}", subdoc.id)
+            
+    # Delete the document itself
+    doc_ref.delete()
+    print(f"Deleted document {document_id} from collection {collection_name}.")
+
+def delete_collection(collection_name):
+    """Delete all documents in a collection(removes the collection), including 
+    their sub-files(subcollections and their documents)."""
+    db = firestore.Client()
+    collection_ref = db.collection(collection_name)
+
+    start_time = time.time()
+    time_limit = 60
+
+    # Loop until no documents are left
+    while True:
```

Comment:
You probably should review [this note](https://cloud.google.com/firestore/docs/manage-data/delete-data#collections) in the docs about deleting a collection. There is a risk if there are too many items in it that can be overcome by batching.

Reply:
I quoted this comment in the ticket for the 'reset_database' ticket. One of the next PRs.